altsrc/yaml_file_loader_test.go

```diff
@@ -1,13 +1,17 @@
-package altsrc_test
+package altsrc
```

Comment:
Lets keep the original package name . we are verifying the uint64 cast types indirectly through tests. No need to verify them separately here.

Reply:
Okay, thanks. resolved it [f8fe5e8](https://github.com/urfave/cli/pull/1669/commits/f8fe5e8769bcf52dcca74897610983eaa56cfa29) and removed the separate uint64 test case.